app/jobs/update_plage_ouvertures_expirations_job.rb

```diff
@@ -0,0 +1,7 @@
+class UpdatePlageOuverturesExpirationsJob < ApplicationJob
+  def perform
+    PlageOuverture.where(expired_cached: false).each do |po|
+      po.update(expired_cached: true) if po.expired?
```

Comment:
actually you could even call `refresh_plage_ouverture_expired_cached` here

`PlageOuverture.where(expired_cached: false).each(&:refresh_plage_ouverture_expired_cached)`